Rethinking_2/End_of_chapter_problems/Chapter 2 - SMALL WORLDS AND LARGE WORLDS.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<p>Remove unused imports (I think pandas and seaborn). Reorder imports see other notebooks as example.</p><p> </p><p>Do this for all notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>